bundles/org.openhab.binding.pjlinkdevice/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pjLinkDevice"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="pjLinkDevice">
+		<label>PJLink Device</label>
+		<description>A PJLink compatible device, e.g. a digital projector</description>
+
+		<channels>
+			<channel id="powerChannel" typeId="powerChannelType" />
+			<channel id="inputChannel" typeId="inputChannelType" />
+			<channel id="audioMuteChannel" typeId="audioMuteChannelType" />
+			<channel id="videoMuteChannel" typeId="videoMuteChannelType" />
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP address</label>
+				<description>The address of the PJLink device to control.</description>
+			</parameter>
+			<parameter name="adminPassword" type="text" required="false">
+				<context>password</context>
+				<label>Password</label>
+				<description>The password of the PJLink device.</description>
+			</parameter>
+			<parameter name="tcpPort" type="integer" required="true">
```

Comment:
As there is a default value and as we can see in your example, this parameter is not required.